packages/auth-server/src/index.ts

```diff
@@ -0,0 +1,65 @@
+import express from 'express';
+import fetch from 'node-fetch';
+
+const app = express();
+app.use(express.json());
+
+const port = 3005;
+
+const GOOGLE_CLIENT_SECRET =
+    process.env.GOOGLE_CLIENT_SECRET || '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'; // TODO: remove temporary fallback
+
+/**
+ * Is server alive?
+ */
+app.get('/status', (_req, res) => {
+    res.send({ status: 'ok' });
+});
+
+/**
+ * Exchange authorization code for refresh token and access token.
+ */
+app.post('/google-oauth-init', async (req, res) => {
+    try {
+        const response = await fetch('https://oauth2.googleapis.com/token', {
+            body: JSON.stringify({
+                code: req.body.code,
+                client_secret: GOOGLE_CLIENT_SECRET,
+                client_id: req.body.clientId,
+                redirect_uri: req.body.redirectUri,
+                grant_type: 'authorization_code',
+                code_verifier: req.body.codeVerifier,
+            }),
+            method: 'POST',
+        });
+        const json = await response.json();
```

Comment:
I would add validation of whether response is in expected format here and if not I would return some anticipated error

Reply:
678e53853979e50eb07e697bec95930df99842d0